src/scss/_brand.scss

```diff
@@ -0,0 +1,35 @@
+@include oBrandDefine('o-overlay', 'master', (
+	'variables': (
+		default-background: oColorsGetPaletteColor('white'),
+		default-border: oColorsGetPaletteColor('black-20'),
+		heading-background: oColorsGetPaletteColor('transparent'),
+		heading-text: oColorsGetPaletteColor('black-80'),
+		shaded-heading-background: oColorsGetPaletteColor('black-80'),
+		shaded-heading-border: oColorsGetPaletteColor('black-80'),
+		shaded-heading-text: oColorsGetPaletteColor('white'),
+		default-close-background: oColorsGetPaletteColor('white'),
+		default-close-text: oColorsGetPaletteColor('black-50'),
+		default-close-border: oColorsGetPaletteColor('black-50'),
+		shaded-close-background: oColorsGetPaletteColor('black-50'),
+		shaded-close-text: oColorsGetPaletteColor('white'),
+		shaded-close-border: oColorsGetPaletteColor('white')
+	)
+));
+
+@include oBrandDefine('o-overlay', 'internal', (
+	'variables': (
+		default-background: oColorsGetPaletteColor('white'),
+		default-border: oColorsGetPaletteColor('transparent'),
+		heading-background: oColorsGetPaletteColor('transparent'),
+		heading-text: oColorsGetPaletteColor('slate'),
```

Comment:
I'm not sure it's appropriate to change the heading text to slate. Unless we have a design that says to? To my eye it looks a little out of place against the darker copy in the overlay.

Otherwise looks good!